systemtests/src/main/java/io/enmasse/systemtest/iot/IoTUtils.java

```diff
@@ -43,9 +22,33 @@
 import io.enmasse.systemtest.utils.TestUtils;
 import io.fabric8.kubernetes.api.model.apps.Deployment;
 import io.fabric8.kubernetes.client.utils.Serialization;
+import io.vertx.core.json.JsonObject;
+import org.assertj.core.api.Condition;
+import org.slf4j.Logger;
 
-public class IoTUtils {
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
 
+import static io.enmasse.systemtest.platform.Kubernetes.getClient;
+import static io.enmasse.systemtest.platform.Kubernetes.iotConfigs;
+import static io.enmasse.systemtest.platform.Kubernetes.iotTenants;
+import static io.enmasse.systemtest.time.TimeMeasuringSystem.Operation.startOperation;
+import static io.enmasse.systemtest.utils.TestUtils.TimeoutHandler.explain;
+import static java.util.Optional.ofNullable;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
we use org.hamcrest.MatcherAssert.assertThat; in other tests so I think you iot should use it as well

Reply:
It is not about the `assertThat`, that just enabled the use of "assertj", which makes a few tests easier later on.